src/validator/CreditCardVendorReading.java

```diff
@@ -0,0 +1,7 @@
+package validator;
+
+import java.util.ArrayList;
+
+public interface CreditCardVendorReading {
+	public ArrayList<CreditCardVendor> returnVendorArray();
```

Comment:
the usual name this this method in Spring Data for example would be `findAll()`. We don't carre if it's an "array" or not (here you return a List, not an array, which is misleading). Also public is not needed. So finally I'd refactor this to:

```suggestion
	Collection<CreditCardVendor> findAll();
```